lib/backend/k8s/k8s.go

```diff
@@ -94,6 +95,9 @@ func NewKubeClient(kc *KubeConfig) (*KubeClient, error) {
 			*override.variable = override.value
 		}
 	}
+	if kc.K8sInsecureSkipTLSVerify {
```

Comment:
Yes indeed, that's why I did it this way.  (After trying the other way first :smile: )

No need for this to be merged soon, though; I only need it for my scale testing work, so best to wait until after we've released Calico 2.1.0.